.env.sample

```diff
@@ -13,3 +13,4 @@ LID_APP_SECRET=
 GOOGLE_APP_ID=
 GOOGLE_APP_SECRET=
 WEBSHOT_URL=
+RESILIENCE_API_KEY=Resilience_0b948b62-94bc-4177-876a-28c0b2b6fb87
```

Comment:
@AdeelKamalMalik - assume there are no issues with making this API key public?

Reply:
@azvoleff That was just a sample key, we will place a new secret key on our severs. But I have removed this sample key as well.